aws_marketplace/using_algorithms/automl/AutoML_-_Train_multiple_models_in_parallel.ipynb

```diff
@@ -0,0 +1,635 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Goal: Train a model using Automl functionality! \n",
+    "\n",
+    "A popular approach to solve a machine learning problem is to try multiple approaches for training a model by running multiple algorithms on a dataset. Based on initial analysis, you can decide which algorithm to use for training and tuning the actual model. However, each algorithm can have specific feature requirements such as data must be numeric, missing values must be addressed before the training, etc. Performing algorithm specific feature engineering tasks can take time. Such a project can be shortened by running an AutoML algorithm that performs feature engineering tasks such as one-hot encoding, generalization, addressing missing values, automatically and then trains models using multiple algorithms in parallel.  \n",
+    "\n",
+    "This notebook demonstrates how to use such an AutoML algorithm fferd by [H2o](https://aws.amazon.com/marketplace/seller-profile?id=55552124-d41b-4bad-90db-72d427682225) in AWS Marketplace for machine learning.  AutoML from H2O trains one or more of following types of models in parallel:\n",
```

Comment:
Please check for typos

s/ffered/offered
s/H2o/H2O

Reply:
Sure.